docs/build/start-building/interacting-with-the-network.md

```diff
@@ -26,23 +26,40 @@ Here are some scripts to set up your own remote networks for development:
 - [Lotus Docker Image](https://github.com/openworklabs/filecoin-docker) for a simple Lotus node Docker container
 - [Filecoin Chart](https://github.com/openworklabs/filecoin-chart) for more complex Lotus node architectures managed with Kubernetes
 
-## Devnet
+## Devnets
 
```

Comment:
A one-liner here for context would be +1, perhaps something like "There are currently two developer networks available for users to try out: the 'Nerpa' devnet, which is recommended for developer audiences, and the Butterfly devnet, which is best suited for miners that are interested in testing out new Lotus features."